src/main/java/com/google/devtools/build/lib/actions/ExecutionRequirements.java

```diff
@@ -118,7 +118,31 @@ public String parseIfMatches(String tag) throws ValidationException {
   }
 
   /** If specified, the timeout of this action in seconds. Must be decimal integer. */
-  public static final String TIMEOUT = "timeout";
+  public static final ParseableRequirement TIMEOUT =
+      ParseableRequirement.create(
+          "timeout:<int>",
```

Comment:
Can we use a syntax that includes a time unit, e.g. `30s` or `5m` or `1h`, for added clarity? (See `DURATION_REGEX` in src/main/java/com/google/devtools/common/options/Converters.java for inspiration.)